Backpack/Chip/Classes/BPKChip.m

```diff
@@ -30,8 +30,9 @@
 @interface BPKChip ()
 @property(nonatomic) BPKLabel *titleLabel;
 @property(nonatomic) CALayer *tintLayer;
-
-@property(nonatomic) UIColor *textColor;
+@property(nonatomic) UIColor *contentColor;
+@property(nullable, nonatomic) UIView *iconView;
```

Comment:
Why isn't this a `BPKIconView` here?

Reply:
We don't do anything to it that a UIView can't do, so it seems unecessary to constrain it to a `BPKIconView`.

Reply:
But it is a `BPIconView`. It's not about what we do it, it's about helping the person reading the code understand. Reading `UIView` I started assuming that we might sometimes use a `BPKIconView` and other times some other type which hurts readability

Reply:
Fixed